helpers/filterSeats.js

```diff
@@ -60,19 +66,69 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections() {
+      var countSections = Object.keys(bandColours).length;
+      // add all section names to an array
+      _.forEach(bandColours, function(bands, section) {
+        bestSections.push(section);
+      });
+      // find the best section
+      bestSections = bestSections.slice(Math.max(countSections - 1, 0)); // just Stalls
+    }
+
+    getBestSections();
+
+    // compare prices
+    function chosenForYou(ticketRate, ticketRateSection) {
+      // ticketRateSection
+      var ticketRateId = ticketRate.ids;
+      var ticketRateColour = ticketRate.colour;
+      var ticketRatePrice = ticketRates[ticketRateId].grossPrice;
+      // check if current ticketRate exists in
+      if (bestSections.indexOf(ticketRateSection) > -1 && bestColours.indexOf(ticketRateColour) > -1) {
+        // check if price is cheaper or exists
+        if (ticketRatePrice < currentPrice || !currentPrice) {
+          // assign ticket to object
+          bestPrice = {
+            id: ticketRateId,
+            price: ticketRatePrice,
+            colour: ticketRateColour
+          };
+        } else {
+          bestPrice = {};
+        }
+      }
+      // get the total amount of tickets set to object
+      var totalTickets = Object.keys(bestPrice).length;
+      // check if any tickets are set
+      if (totalTickets > 0) {
+        // sort best tickets by price then by colour
+        bestPrice = _.sortBy(bestPrice, ['price', 'colour']);
```

Comment:
The Lodash sortBy wouldn't work for colour since this is a string value and will be sorted in alphabetical order. We can't rely on this especially if we're going to introduce also the platinum colour. So my suggestion is to map colours to numbers in some way so that we can sort them according to their "weight".